vent/common/values.py

```diff
@@ -318,3 +318,4 @@ def __getitem__(self, key):
 whether (``True``) or not (``False``) a value is allowed.
 """
 
+MAX_STACK_DEPTH = 20
```

Comment:
Better to put this in logging.py, as this only related to logging.

Reply:
Ya I think there will need to be a separate user/system-configuration file at some point for stuff like draw speed, ip:port combinations, etc.

Reply:
but agreed with this until then^^